### DISCUSSION TITLE
Simple list

### DISCUSSION THREAD

**Discussion by anapego** [TiendaNube/nimbus-design-system#179] [Q&A] [answered]
**Ajustar componente e template de lista:**

![Captura de Tela 2023-07-03 às 15 13 53](https://github.com/TiendaNube/nimbus-design-system/assets/92194136/356e35aa-eac1-488a-a086-5b64b281e3c3)

**Comment by juanchigallego** [comment, marked as answer]
Oi @anapego ! O issue ja foi ressolvido com a ultima atualização da biblioteca. Pode atualizar a lib para ter o fix. Qualquer dúvida pode falar com a gente!
